invenio_communities/assets/semantic-ui/js/invenio_communities/subcommunity/new.js

```diff
@@ -124,7 +124,7 @@ class CommunityCreateForm extends Component {
     try {
       const response = await this.cancellableCreate.promise;
       setSubmitting(false);
-      window.location.href = response.data.links.settings_html;
+      window.location.href = response.data.links.self_html;
```

Comment:
`self_html` returns the request URL (`/requests/<uuid>`) but it doesn't work. The URL that works is `/communities/eu/requests/<uuid>`. 

I am not sure why this happens but I am investigating it